build.sbt

```diff
@@ -111,7 +112,6 @@ lazy val common = project
   .settings(
     libraryDependencies ++=
       LucumaSSO.value ++
-        LucumaBC.value ++
```

Comment:
We also need to remove broadcast-channel from `package.json` but I currently couldn't do anything with `npm` while FA is borked. Maybe someone with it cached can run `npm uninstall broadcast-channel`.